### PR TITLE
Update GeForce.com.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: src/chrome/content/rules/GeForce.com.xml [hunk 1]
```diff
@@ -1,51 +1,28 @@
 <!--
 	For other nVidia coverage, see NVidia.xml.
```
Comment (J0WI) [inline src/chrome/content/rules/GeForce.com.xml]: There is a bunch of language subdomains that does not match the given certificate, like `cn.geforce.com`, `fr.geforce.com`, `uk.geforce.com` etc.

File: src/chrome/content/rules/GeForce.com.xml [hunk 2]
```diff
@@ -1,51 +1,28 @@
...
+	<target host="www.stagegeforce.geforce.com" />
+	<target host="chn.stagegeforce.geforce.com" />
+	<target host="gbr.stagegeforce.geforce.com" />
+	<target host="gfwsl.stagegeforce.geforce.com" />
+	<target host="twn.stagegeforce.geforce.com" />
+	<target host="demo.uk.geforce.com" />
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^http://geforce\.com/" to="https://www.geforce.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/GeForce.com.xml]: Please add a comment for the issue on `^`.

File: src/chrome/content/rules/GeForce.com.xml (hunk 1 above)
Comment (koops76) [inline src/chrome/content/rules/GeForce.com.xml]: I don't have them rewritten.

Comment (J0WI) [inline src/chrome/content/rules/GeForce.com.xml]: But could you add a comment about this behavior?

File: src/chrome/content/rules/GeForce.com.xml (hunk 2 above)
Comment (koops76) [inline src/chrome/content/rules/GeForce.com.xml]: @J0WI Done.

File: src/chrome/content/rules/GeForce.com.xml (hunk 1 above)
Comment (J0WI) [inline src/chrome/content/rules/GeForce.com.xml]: I think we can also remove the `(partial)` flag.

Comment (J0WI) [reply src/chrome/content/rules/GeForce.com.xml]: ping @koops76

Comment (koops76) [reply src/chrome/content/rules/GeForce.com.xml]: @J0WI Done.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**